libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/testHelpers.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2020 GoodData Corporation
+import { IImplicitDrillDown } from "../../interfaces/Visualization";
+import { IDrillEventIntersectionElement, IDrillEvent, VisType } from "@gooddata/sdk-ui";
+import { IAttribute, IInsight, IInsightDefinition, uriRef } from "@gooddata/sdk-model";
```

Comment:
if these are test helpers, why are they not under `test` directory?

Reply:
Good point I will move it

Reply:
Done